aws/templates/solution/solution_lb.ftl

```diff
@@ -20,6 +20,13 @@
 
             [#assign engine = solution.Engine]
 
+            [#assign healthCheckPort = "" ]
+            [#if engine == "classic" && solution.HealthCheckPort?has_content ]
+                [#assign healthCheckPort = ports[solution.HealthCheckPort]]
+            [#else]
+                [@cfPreconditionFailed listMode "solution_lb" {} "No health check port provided" /]    
```

Comment:
Is this logic right? I  thought the "HealthCheckPort was only added to support the classic lb but this code will catch all lb types?